src/Microsoft.Health.Dicom.SqlServer/Features/Query/SqlQueryGenerator.cs

```diff
@@ -331,6 +331,24 @@ public override void Visit(DoubleSingleValueMatchCondition doubleSingleValueMatc
                 .AppendLine();
         }
 
+        public override void Visit(LongRangeValueMatchCondition rangeValueMatchCondition)
```

Comment:
I am using you are not checking on UTC value and only supporting literal value

Reply:
For now, we are not supporting offsets in QIDO. Once we work on implementing this feature, that's when UTCValue will come in handy. For now, we are simply storing it such that we have enough data to work on whenever we support this in future.